pkg/controller/pxcbackup/controller.go

```diff
@@ -365,6 +363,13 @@ func (r *ReconcilePerconaXtraDBClusterBackup) runS3BackupFinalizer(cr *api.Perco
 
 func removeBackup(bucket, backup string, s3cli *minio.Client) func() error {
 	return func() error {
+		// this is needed to understand if user provided some path
+		// on s3, instead of just a bucket name
+		bucketSplitted := strings.Split(bucket, "/")
```

Comment:
asking just to be sure: we're deleting the `s3://` part before here, aren't we?

Reply:
yea, here we have "raw" bucket name, without an `s3://` part